optaplanner-benchmark/pom.xml

```diff
@@ -28,10 +28,6 @@
 
   <dependencies>
     <!-- Internal dependencies -->
-    <dependency>
```

Comment:
There will be a conflict with https://github.com/kiegroup/optaplanner/pull/824

Reply:
Yes, unfortunate. Feel free to add to tomorrow's meeting if there's way to avoid this.